delivery_carrier_label_default/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2013-2018 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+{
+    "name": "Default label for carrier labels",
+    "summary": ("This module defines a basic label to print "
+                "when no specific carrier is selected."),
+    "version": "11.0.1.0.0",
+    "development_status": "Beta",
```

Comment:
is it really beta?

Reply:
I have not enough perspective on that point, there was no development status before, so I preferred to be conservative about this status at first, but if you think it's `Production/Stable`, so be it. What do you think?